packages/redhat/Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN yum install -y \
         rpmlint \
         yum-utils \
         && \
+    yum install -y \
```

Comment:
Why running a new `yum install -y` when you could simply add `alien` to the list of installed package by the existing `yum install -y`?

Reply:
Because during my tests, when I add `alien` with all other packages, it doesn't work.
I think it's a problem of dependencies uninstalled until all the previous packages are installed.
Add a new line with a new `yum install -y` is the only way i found to install `alien`

Reply:
`alien` sits in EPEL, not in the core repositories. Since we only install `epel-release` (which configures the EPEL repository on the system) in the first `yum install` invocation. Before that, EPEL is not available, so trying to install something from it fails.

So, not an issue and the second `yum install` invocation is fine (heck, there's no other way).

Reply:
Weird, what kind of error do you get?

Reply:
Ho ok, makes sense.